.eslintrc.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+    plugins: [
+      "prettier"
+    ],
+    overrides: [
```

Comment:
@roboshoes I’ve separated TS code from generic Node code.

Reply:
Awesome. That makes sense.